src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -0,0 +1,255 @@
+  @Mock private HttpServletResponse response;
+
+  private TranscriptServlet servlet;
+
+  private LocalDatastoreServiceTestConfig datastoreConfig =
+      (new LocalDatastoreServiceTestConfig()).setNoStorage(true);
+  private final LocalServiceTestHelper localServiceHelper =
+      new LocalServiceTestHelper(datastoreConfig);
+  private DatastoreService datastore;
+  private Gson gson;
+  StringWriter content;
+
+  private static final String LECTURE_ID_A = "123";
+  private static final String LECTURE_ID_B = "345";
+  private static final String LECTURE_ID_C = "234";
+  private static final String SHORT_VIDEO_ID = "Obgnr9pc820";
+  private static final String LONG_VIDEO_ID = "jNQXAC9IVRw";
+  private static final String SHORT_VIDEO_JSON =
+      "[{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":1},"
+      + "\"lectureKey\":{\"kind\":\"lecture\",\"id\":123},"
+      + "\"start\":\"0.4\",\"duration\":\"1\",\"content\":\" \"},"
+      + "{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":2},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"2.28\",\"duration\":\"1\",\"content\""
+      + ":\"Hi\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":3},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"5.04\",\"duration\":\"1.6\","
+      + "\"content\":\"Okay\"}]";
+  private static final String LONG_VIDEO_JSON =
+      "[{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":1},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"1.3\",\"duration\":"
+      + "\"3.1\",\"content\":\"All right, so here we are\\nin front of the "
+      + "elephants,\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":4},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"12.7\",\"duration\":\"4.3\",\"content\":"
+      + "\"and that&#39;s, that&#39;s cool.\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":5},"
+      + "\"lectureKey\":{\"kind\":\"lecture\",\"id\":123},\"start\":\"17\",\"duration\":\"1.767\",\""
+      + "content\":\"And that&#39;s pretty much all there is to say.\"},{\"transcriptKey\":{\"kind\""
+      + ":\"TranscriptLine\",\"id\":2},\"lectureKey\":{\"kind\":\"lecture\",\"id\":123},\"start\":"
+      + "\"4.4\",\"duration\":\"4.766\",\"content\":\"the cool thing about these guys\\nis that "
+      + "they have really,\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":3},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"9.166\",\"duration\":\"3.534\","
+      + "\"content\":\"really, really long trunks,\"}]";
+
+  @Before
+  public void setUp() throws ServletException, IOException {
+    localServiceHelper.setUp();
+    servlet = new TranscriptServlet();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    servlet.init(datastore);
+    gson = new GsonBuilder().registerTypeAdapterFactory(GenerateTypeAdapter.FACTORY).create();
+    content = new StringWriter();
```

Comment:
nit: rename content as lectureTranscript? content is a bit vague of a variable name.

Reply:
done!